lineman/app/services/key_event_service.coffee

```diff
@@ -28,5 +29,5 @@ angular.module('loomioApp').factory 'KeyEventService', ($rootScope) ->
     submitOnEnter: (scope) ->
       @registerKeyEvent scope, 'pressedEnter', scope.submit, (active, event) =>
         (event.ctrlKey or event.metaKey) and
-        angular.element(active).scope() == scope and
+        angular.element(active).scope().$$isolateBindings == scope.$$isolateBindings and
```

Comment:
Because things like `<form>` and `<textarea>` are directives in angular, this scope comparison wasn't working. What I really want is `scope.isChildOf(angular.element(active).scope())`, but couldn't find that anywhere and don't want to write my own.

(This check is there at all in order to prevent you from, say, submitting your comment while creating a new thread in a model)